src/renderer/reducers/deposit.ts

```diff
@@ -0,0 +1,36 @@
+import {IGenerateDepositAction, IVerifyDepositAction} from "../actions";
+import {DepositActionTypes} from "../constants/action-types";
+import {Action} from "redux";
+
+export interface IDepositState {
+    isDepositGenerated: boolean;
+    isDepositVisible: boolean;
+    txData: string;
+}
+
+const initialState: IDepositState = {
+    isDepositGenerated: false,
+    isDepositVisible: false,
```

Comment:
Seems like it would be more useful to be amount deposited as <32 amounts will need special status in dashboard that they need more eth